_scripts/teardown.sh

```diff
@@ -0,0 +1,2 @@
+docker-compose down --volumes --remove-orphans
+docker volume prune --force
```

Comment:
Interesting!  Never thought to include the second line as a backup teardown process in case the first line didn't work.  Might try that in future.  :)